frontend/src/modules/dashboard/views/HFNCGraphView.tsx

```diff
@@ -159,34 +159,13 @@ const HFNCMainView = (): JSX.Element => {
       <Grid container item xs={12} alignItems="stretch" className={classes.topPanel}>
         <Grid item xs={4} alignItems="stretch" className={classes.topLeftPanel}>
           <Grid container item justify="center" alignItems="stretch">
-            <ValueInfo
-              mainContainer={{
-                selector: getSmoothedSpO2,
-                label: 'SpO2',
-                stateKey: 'spo2',
-                units: PERCENT,
-              }}
-            />
+            <ValueInfo selector={getSmoothedSpO2} label="SpO2" stateKey="spo2" units={PERCENT} />
```

Comment:
I really like this approach of parameterizing the `ValueInfo` component on the selector as an argument, this way we don't have to create a bunch of files representing each combination of `ValueInfo` and selector.